src/griffe/__init__.py

```diff
@@ -16,6 +16,8 @@
 from griffe.loader import load
 from griffe.logger import get_logger
 
+__version__ = "0.32.3"
```

Comment:
I'm not a fan of hardcoding the version in the package itself.
Could we use `importlib.metadata.version` instead?
See example in my next comment.

Reply:
`importlib.metadata.version` sounds good to me. Added. Thanks for the example. Much appreciated!